daml.yaml

```diff
@@ -1,12 +1,12 @@
-sdk-version: 0.13.55
+sdk-version: 1.0.0
 name: ex-tutorial-nodejs
 source: daml/PingPong.daml
 parties:
 - Alice
 - Bob
 exposed-modules:
```

Comment:
I would recommend to remove the `exposed-modules` field. It already causes problems and warnings in some cases and not putting it there is a safe choice (it exposes everything).

Reply:
Thanks!

Reply:
Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/12/commits/524cd550328bfca79db7c7b6a6c6d0fb113ffdb9